app/src/marketplace/containers/ProductController/useProductPermissions.jsx

```diff
@@ -45,16 +46,20 @@ function usePermissionContextValue() {
     }, [productId, loadedOnce, isPending, loadPermissions])
 
     const hasPermissions = !!permissions
+    /* eslint-disable camelcase */
```

Comment:
`eslint-disable camelcase` doesn't seem to be necessary

Reply:
👍 fixed.